### PR TITLE
Include notice about WP 5.7

### PR DESCRIPTION
**Pull request by bordoni**
🎫 [TCMN-95]
---

Screen Shot 2021-01-13 at 9 50 36 PM

[TCMN-95]: https://theeventscalendar.atlassian.net/browse/TCMN-95

### CODE REVIEW

File: src/Tribe/Admin/Notice/WP_Version.php [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+/**
+ * Various WordPress version notices.
+ *
+ * @since TBD
+ */
+
+use \Tribe__Date_Utils as Dates;
+use Tribe__Main as Common;
+
+class WP_Version {
```
Comment (paulmskim) [inline src/Tribe/Admin/Notice/WP_Version.php:13]: should this comment be above the class declaration?

**Review by paulmskim — COMMENTED** [review]
code looks good, just one question

File: src/Tribe/Admin/Notice/WP_Version.php [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+	 */
+	public function wp_version_57_display_notice() {
+		global $wp_version;
+		$is_wp_57 = version_compare( $wp_version, '5.7-beta', '>=' );
+		$html = '';
+
+		if ( $is_wp_57 ) {
+			$html .= esc_html__( 'You are using WordPress 5.7 which included a major jQuery update that may cause compatibility issues with past versions of The Events Calendar, Event Tickets and other plugins.', 'tribe-common' );
+		} else {
+			$html .= esc_html__( 'WordPress 5.7 includes a major jQuery update that may cause compatibility issues with past versions of The Events Calendar, Event Tickets and other plugins.', 'tribe-common' );
```
Comment (Camwyn) [inline src/Tribe/Admin/Notice/WP_Version.php:68]:
```suggestion
			$html .= esc_html__( 'WordPress 5.7 included a major jQuery update that may cause compatibility issues with past versions of The Events Calendar, Event Tickets and other plugins.', 'tribe-common' );
```

File: src/Tribe/Admin/Notice/WP_Version.php [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+	 *
+	 * @since TBD
+	 *
+	 * @return boolean
+	 */
+	public function wp_version_57_should_display() {
+		global $wp_version;
+
+		$wp_version_min_version_required = '5.8';
+		$common_version_required = '4.12.18-dev';
```
Comment (Camwyn) [inline src/Tribe/Admin/Notice/WP_Version.php:44]: We're expecting to bump this as needed? It's max 3 common releases away?

**Review by Camwyn — COMMENTED** [review]
Question about versions, test suggestion, code looks good.

File: src/Tribe/Admin/Notice/WP_Version.php (hunk 2 above)
Comment (Camwyn) [reply src/Tribe/Admin/Notice/WP_Version.php:68]: This will show on versions past 5.7, so past tense (or do something like "WordPress 5.7 and on..."

**Review by Camwyn — APPROVED** [review]

File: src/Tribe/Admin/Notice/WP_Version.php (hunk 2 above)
Comment (bordoni) [reply src/Tribe/Admin/Notice/WP_Version.php:68]: Sorry this is for versions before 5.7

**Review by sc0ttkclark — APPROVED** [review]

**Pull request merged**